testing/integration/otel_test.go

```diff
@@ -1580,9 +1574,8 @@ service:
 				require.True(t, found, "expected message field in document %q", hit.Source)
 				msg, ok := message.(string)
 				require.True(t, ok, "expected message field to be a string, got %T", message)
-				if _, found := uniqueIngestedLogs[msg]; found {
-					t.Logf("log line %q was ingested more than once", message)
-				}
+				_, found = uniqueIngestedLogs[msg]
+				require.False(t, found, "found duplicated log message %q", msg)
```

Comment:
Any specific reason why we can't use [assert.NotContains](https://pkg.go.dev/github.com/stretchr/testify/assert#NotContains) here?
```suggestion
				require.NotContainsf(uniqueIngestedLogs, msg, "found duplicated log message %q", msg)
```

Reply:
Not at all, I just didn't knew about this function. TIL.